### ISSUE TITLE
Metatarsal head pad

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11677]
# Metatarsal head pad

## Proposed definition

An abnormally prominent soft tissue pad overlying the head of a metatarsal bone on the plantar surface of the foot.

## Parent term

- `HP:0100872 Abnormality of the plantar skin of foot`

## Synonyms

- Plantar metatarsal head pad

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648
Adding new term: 
- ID: HP:6001518
- label: Metatarsal fat pad
- parents: [Abnormality of the plantar skin of foot (HP:0100872)](https://hpo.jax.org/app/browse/term/HP:0100872)
- pmid: PMID:41464288
- def: An abnormally prominent soft tissue pad overlying the head of a metatarsal bone on the plantar surface of the foot.
- comment: The metatarasal fat pad is a connective and adipose layer located between the reticular dermis and deeper structures such as the plantar fascia, muscles, and bones. In the forefoot, beneath the metatarsal heads, the plantar fat pad is organized into closed compartments with a honeycomb-like structure, where adipocytes are surrounded by fibrous septa rich in elastin.
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11677
- synonyms: Metatarsal head pad(NONE)
